arduino/CAN Sample Code/CAN2/CAN2_RECEIVER/CAN2_RECEIVER.ino

```diff
@@ -0,0 +1,54 @@
+#include <mcp_can.h>    // ADD library https://github.com/coryjfowler/MCP_CAN_lib
+#include <SPI.h>
+
+// Custom SPI Pins for MCP2515
+#define CAN2_CS_PIN    GPIO_NUM_10
+#define CAN2_SPI_SCK   GPIO_NUM_12
+#define CAN2_SPI_MISO  GPIO_NUM_13
+#define CAN2_SPI_MOSI  GPIO_NUM_11
+
+SPIClass CAN2_SPI(HSPI);  // HSPI bus
+MCP_CAN CAN2(&CAN2_SPI, CAN2_CS_PIN);
+
+bool setupCAN2()
+{
+  CAN2_SPI.begin(CAN2_SPI_SCK, CAN2_SPI_MISO, CAN2_SPI_MOSI, CAN2_CS_PIN);
+
+  if (CAN2.begin(MCP_STDEXT, CAN_500KBPS, MCP_16MHZ) == CAN_OK) {
+    CAN2.setMode(MCP_NORMAL);
+    return true;
+  } else {
+    return false;
+  }
+}
+
+void setup()
+{
+  Serial.begin(115200);
+  while (!Serial);
+
+  if (!setupCAN2()) {
+    Serial.println("Failed to initialize CAN2. Check connections.");
+    while (1);
+  } else {
+    Serial.println("MCP2515 Initialized Successfully (Receiver)");
+  }
+}
+
+void loop()
+{
+  if (CAN2.checkReceive() == CAN_MSGAVAIL) {
```

Comment:
like you did with CAN1 receiver, create a function that receives the message and prints it out. 

```
  CAN2_readMsg(); // read CAN2 message
```